### PR TITLE
Literate for state-feedback abstractions

### PR DESCRIPTION
**Pull request by egidioln**
Added the literate for state-feedback abstractions of PWA systems

Preview https://dionysos-dev.github.io/Dionysos.jl/previews/PR181/

### CODE REVIEW

File: docs/src/examples/State-feedback Abstraction: PWA System.jl [hunk 1]
```diff
@@ -0,0 +1,352 @@
+using Test     #src
+# # Example: Optimal control of a PWA System by State-feedback Abstractions
+#
+#md # [![Binder](https://mybinder.org/badge_logo.svg)](@__BINDER_ROOT_URL__/generated/State-feedback Abstraction: PWA System.ipynb)
+#md # [![nbviewer](https://img.shields.io/badge/show-nbviewer-579ACA.svg)](@__NBVIEWER_ROOT_URL__/generated/State-feedback Abstraction: PWA System.ipynb)
```
Comment (blegat) [inline docs/src/examples/State-feedback Abstraction: PWA System.jl:5]: Add an empty line between badges and the rest (see html output: https://dionysos-dev.github.io/Dionysos.jl/previews/PR181/generated/State-feedback%20Abstraction:%20PWA%20System/)

File: docs/src/examples/State-feedback Abstraction: PWA System.jl [hunk 2]
```diff
@@ -0,0 +1,352 @@
...
+
+domainU = domainX; # input space
+Dionysos.Domain.add_set!(domainU, rectU, Dionysos.Domain.OUTER)
+
+#and the symbolic model for the state-feedback abstraction $\tilde{\mathcal{S}}$
+symmodel = Dionysos.Symbolic.NewSymbolicModelListList(domainX, domainU);
+empty!(symmodel.autom)
+
+
+# Now let us define the L matrix defining the stage cost $\mathcal{J}(x,u) = ||L*[x; u ; 1]||^2_2$
```
Comment (blegat) [inline docs/src/examples/State-feedback Abstraction: PWA System.jl:107]:
```suggestion
# Now let us define the L matrix defining the stage cost $\mathcal{J}(x,u) = ||L \cdot [x; u ; 1]||^2_2$
```

File: docs/src/examples/State-feedback Abstraction: PWA System.jl [hunk 3]
```diff
@@ -0,0 +1,352 @@
...
+cmap = PyPlot.ColorMap("mycolor",hcat([0.0,0.8,0.5,0.5],[0.8,0.0,0.5,0.5])');
+PyPlot.colorbar(PyPlot.ScalarMappable(norm=PyPlot.cm.colors.Normalize(vmin=0, vmax=lyap_max),cmap=cmap),shrink=0.7)
+
+PyPlot.xlabel("\$x_1\$", fontsize=14)
+PyPlot.ylabel("\$x_2\$", fontsize=14)
+PyPlot.title("Trajectory and Lyapunov-like Fun.", fontsize=14)
+gcf() #md
+gcf() #nb
+
+# We recall that, to speed up the build time of this documentation, some values were modified in comparison with [1, Example 2]. To obtain the sabe figures use `Usz = 50`, `Wsz = 5` and `n_step = 5`.
```
Comment (blegat) [inline docs/src/examples/State-feedback Abstraction: PWA System.jl:343]:
```suggestion
# We recall that, to speed up the build time of this documentation, some values were modified in comparison with [1, Example 2]. To obtain the same figures use `Usz = 50`, `Wsz = 5` and `n_step = 5`.
```

**Review by blegat — APPROVED** [review]

**Comment by egidioln** [comment]
Thanks @blegat ! :)

**Pull request merged**